sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/azure/fix.adoc

```diff
@@ -0,0 +1,36 @@
+== Why is this an issue?
+
+Hardcoding secret is one of the most widespread causes of secret leaks. It happens
```

Comment:
Please apply the feedback from the [common PR](https://github.com/SonarSource/sonar-text/pull/126) here as well.